tests/test_compressors.py

```diff
@@ -0,0 +1,81 @@
+import os
+import unittest
+from pathlib import Path
+
+from numcodecs import Blosc
+from wavpack_numcodecs import WavPack
+
+from transfer.compressors import EphysCompressors
+from transfer.readers import EphysReaders
+
+TEST_DIR = Path(os.path.dirname(os.path.realpath(__file__)))
+RESOURCES_DIR = TEST_DIR / "resources"
+
+
+class TestEphysCompressors(unittest.TestCase):
+    def test_get_compressor(self):
+        blosc_configs = {
+            "cname": "zstd",
+            "clevel": 9,
+            "shuffle": Blosc.BITSHUFFLE,
+        }
+        wavpack_configs = {"level": 3}
+        blosc = EphysCompressors.get_compressor(
+            EphysCompressors.Compressors.blosc.name, blosc_configs
+        )
+        wavpack = EphysCompressors.get_compressor(
+            EphysCompressors.Compressors.wavpack.name, wavpack_configs
+        )
+        expected_blosc = Blosc(
+            cname="zstd", clevel=9, shuffle=Blosc.BITSHUFFLE
+        )
+        expected_wavpack = WavPack(level=3)
+        self.assertEqual(blosc, expected_blosc)
+        self.assertEqual(wavpack, expected_wavpack)
+
+    def test_get_compressor_fails(self):
+        with self.assertRaises(Exception):
+            EphysCompressors.get_compressor("Made up name")
+
+    def test_scale_recording(self):
+        open_ephys_dir = (
+            RESOURCES_DIR / "v0.6.x_neuropixels_multiexp_multistream"
+        )
+        openephys_reader = EphysReaders.Readers.openephys.name
+        read_blocks = EphysReaders.get_read_blocks(
+            openephys_reader, open_ephys_dir
+        )
+
+        read_block = next(read_blocks)
+        expected_lsb_value = 7
```

Comment:
@jtyoung84 for this specific recording the LSB is actually 3, but since we are only using a `chunk_size` of 20, it gets wrongly estimated

Reply:
@alejoe91 if I set the chunk_size to `chunk_size = min(read_block['recording'].get_num_frames(0)-1, 10000)`, which I think is 99, then the LSB value is 4. Does that make sense?

Reply:
It makes more sense! The lsb on the full recording is 3..but maybe 100 frames is too short to properly estimate it

Reply:
When you say full recording, do you mean the recording that the small data set in the test resources is pulled from?

Reply:
exactly